resources/servo.css

```diff
@@ -60,3 +60,11 @@ details[open]::-servo-details-content {
   display: block;
 }
 
+/*
+ * Until servo supports svg properly, make sure to at least prevent svg
+ * children from being layed out and rendered like usual html.
+ * https://github.com/servo/servo/issues/10646
+ */
+svg > * {
+  display: none;
```

Comment:
The namespaced rule does not match correctly for the testcase. :-(
Does that mean that the svg element is not correctly auto-namespaced?